driver.go

```diff
@@ -176,3 +178,14 @@ func connect(opts *Options) (*Conn, error) {
 
 	return &Conn{client: client, t: transport, log: logger}, nil
 }
+
+func buildUserAgentSuffix(clientId string) string {
+	strs := []string{runtime.Version(), runtime.GOOS, clientId}
```

Comment:
here https://github.com/databricks/databricks-sql-go/pull/21#discussion_r966468166 we agreed to do runtime version and name outside of the scope of the PR. we may need to discuss the format, etc.

Reply:
Sure I'll comment it out. Just wanted to keep for reference

Reply:
Actually let me ask, if something is partially implemented is it okay to comment it out or should I remove all together? I'm not sure what the protocol is for open source

Reply:
for this specific case. I would just remove it. you always have the commit history so you can copy/paste it back.
this is not a big deal either way.

Reply:
Sounds good, will remove since it'll be in this current commit.